src/lib/services/api-token-service.ts

```diff
@@ -76,7 +78,6 @@ export class ApiTokenService {
         this.eventService = eventService;
         this.environmentStore = environmentStore;
         this.logger = config.getLogger('/services/api-token-service.ts');
-        this.fetchActiveTokens();
```

Comment:
No need to call it here, probably updateLastSeen below is also pointless because it probably does nothing:
https://github.com/Unleash/unleash/blob/3bea6bbd204098ec04cecd8000dfa060a9e5314f/src/lib/services/api-token-service.ts#L101-L102